TeamSnapSDK/SDK/DataTypes/TSDKTeamFee.m

```diff
@@ -16,4 +16,12 @@ + (NSString *)SDKType {
     return @"team_fee";
 }
 
+- (NSString *)teamFeeDescription {
```

Comment:
These shouldn't be needed, but there was a bug in the SDK that I fixed as part of the Team Media branch that caused the "description" postfix not to be handled properly on multi word types. The SDK tries to auto convert API objects sent as "description" to camelCaseClassDescription. 

https://github.com/teamsnap/teamsnap-SDK-iOS/commit/c255bf220ce6955f25188ddde914b48619b35d9a

You can either cherry pick that PR, or we'll need to remember to remove these two methods after my stuff is merged.